### PR TITLE
Fixed the new variable name in Prepare> Column:Text> Split dialog

### LINKED ISSUE #8085: Correct the new variable name in the Prepare > Column: Text > Split dialogue
@N-thony I assume this will be an easy one to allocate?

image

Here is the split dialogue and here is the data set:

[PGStation.zip](https://github.com/africanmathsinitiative/R-Instat/files/10516286/PGStation.zip)

I am splitting the same variable twice (or more).
a) I first split on a space - and made the prefix into `morning`.
b) Then I split again on the morning variable using colon for the split.  I called it hour, but it insists it likes the name morning. 

That's the problem.  I have tried multiple times and it keeps liking the name morning once it has started!

### PR DESCRIPTION
**Pull request by anastasia-mbithe**
This fixes #8085

@rdstern and @N-thony, this is ready for testing.

### CODE REVIEW

**Comment by berylwaswa** [comment]
I have tested the split dialog with the data provided by Roger and it works fine now.

Here is the data set:

[PGStation.zip](https://github.com/africanmathsinitiative/R-Instat/files/10516286/PGStation.zip)

I split the same variable thrice.
a) I first split on a space - and made the prefix into morning, and this worked fine.
b) Then I split again on the morning variable using the colon for the split. I called it hour, and it was split into hour and hour1.
c) Then I split again the morning variable using semi-colon for the split. I called it min, and it was split into min and min1.
 
It now changes the name from morning to the given prefixes.

File: instat/dlgSplitText.vb [hunk 1]
```diff
@@ -126,7 +126,7 @@ Public Class dlgSplitText
         clsTextComponentsMaximum.AddParameter("n", "Inf", iPosition:=2)
         clsTextComponentsMaximum.AddParameter("simplify", "TRUE", iPosition:=3)
 
-        clsTextComponentsFixed.SetAssignTo(strTemp:="split", strTempDataframe:=ucrSelectorSplitTextColumn.ucrAvailableDataFrames.cboAvailableDataFrames.Text, strTempColumn:="split", bAssignToIsPrefix:=True)
+        clsTextComponentsFixed.SetAssignTo(ucrSaveColumn.GetText(), strTempDataframe:=ucrSelectorSplitTextColumn.ucrAvailableDataFrames.cboAvailableDataFrames.Text, strTempColumn:=ucrSaveColumn.GetText)
```
Comment (lloyddewit) [inline instat/dlgSplitText.vb:129]: make consistent?
```suggestion
        clsTextComponentsFixed.SetAssignTo(ucrSaveColumn.GetText(), strTempDataframe:=ucrSelectorSplitTextColumn.ucrAvailableDataFrames.cboAvailableDataFrames.Text, strTempColumn:=ucrSaveColumn.GetText())
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@anastasia-mbithe Thank you, this looks good to me.

@N-thony @berylwaswa May I merge this? Or do you prefer to wait and see if @rdstern has time to test/approve?

@rdstern we (@N-thony @berylwaswa and I) discussed the testing approach this morning. We agreed that, during the sprint, for lower risk PRs, that we may go ahead and merge without your test/approval. You would have the option to test later, and we could raise issues for any problems found (and fix these before the Feb build). I hope that's OK. If you prefer a different approach, then please let us know.

Thanks

**Comment by rdstern** [comment]
Perfect on the merging this week.

**Pull request merged**